specifyweb/frontend/js_src/lib/components/WbPlanView/Mapper.tsx

```diff
@@ -281,6 +282,18 @@ export function Mapper(props: {
     mappingPathIsComplete(state.mappingView) &&
     getMappedFieldsBind(state.mappingView).length === 0;
 
+  const handleDeleteLine = (line: number): void => {
+    dispatch({
+      type: 'UpdateLinesAction',
+      lines: state.lines.filter((_, index) => index !== line),
+    });
+    props.onDeleteLine(line, props.baseTableName);
+    dispatch({
```

Comment:
Remove this action -> modify `UpdateLinesAction` to do this
Otherwise it's a bit bug-prone (adds a possibility of developers changing the lines, but forgetting to mark it as "changes made")

Or if you want to make it even better, get rid of the "changesMade" state entirely, and instead compute it dynamically, like it is in the other places based on storing the initial state in a ref and on state changes comparing if lines are the same

Reply:
if I do this: 
`  function handleDeleteLine(line: number) {
    dispatch({
      type: 'UpdateLinesAction',
      lines: removeItem(state.lines, line),
      changesMade: true,
    });
    props.onDeleteLine(line, props.baseTableName);
  }`
  The delete button seems to have no effect